airflow/config_templates/config.yml

```diff
@@ -1032,18 +1032,6 @@ logging:
       type: boolean
       example: ~
       default: "False"
-    enable_task_context_logger:
```

Comment:
Could you add a [newsfragment](https://github.com/apache/airflow/tree/main/newsfragments), please?

Reply:
yes